frontend/src/lib/components/LemonRow/LemonRow.tsx

```diff
@@ -34,6 +34,7 @@ export interface LemonRowPropsBase<T extends keyof JSX.IntrinsicElements>
     /** A compact row is slightly smaller than normal to better look inline with text. */
     compact?: boolean
     'data-attr'?: string
+    size?: 'small' | 'medium' | 'large'
```

Comment:
We already have prop `compact` (which is pretty much `size="small"`) and should definitely avoid having two props for size. Can we unify those? Preferably preserving existing `compact` styling to avoid breaking things.

Reply:
Ahh I didn't see that `compact` prop. Just did moved this refactor to a separate PR, so it's a bit easier to read. https://github.com/PostHog/posthog/pull/9643